src/Microsoft.ML.Data/Utilities/LocalEnvironment.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.ML.Runtime.Data
+{
+    using Stopwatch = System.Diagnostics.Stopwatch;
+
+    /// <summary>
+    /// An ML.NET environment for local execution.
+    /// </summary>
+    public sealed class LocalEnvironment : HostEnvironmentBase<LocalEnvironment>
```

Comment:
We should have some tests that cover this class.

Reply:
I made API scenario tests use it.

---
In reply to: [218093190](https://github.com/dotnet/machinelearning/pull/923#discussion_r218093190) [](ancestors = 218093190)